package.json

```diff
@@ -3,6 +3,9 @@
   "version": "2.0.0",
   "description": "Webpack loader that adjusts source maps",
   "main": "index.js",
+  "engines": {
+    "node": ">=10"
```

Comment:
@TrySound IIRC `resolve-url-loader` is at node 8. I imagine this is not compatible with minimum requirements there.

Reply:
This change is breaking anyway. resolve-url-loader may drop node 8 support as well in v4.

Reply:
lol. Wasn't planning to. v4 is pretty much ready to go, just needs docs.

Reply:
I'm thinking if you match these deps to those proposed for `resolve-url-loader` on the `v4-development` branch that will be a good PR.

Reply:
Ok. We can use camel-case instead which does not have node restrictions

Reply:
Resolved by #15